ui/DesignSystem/Component/Wallet/SendReceive.svelte

```diff
@@ -1,8 +1,21 @@
 <script>
+  import { push } from "svelte-spa-router";
+  import * as path from "../../../src/path.ts";
+  import { fromStore, toStore, amountStore } from "../../../src/transfer.ts";
+
   import { Button, Input, Icon, Title } from "../../Primitive";
   import Receive from "./Receive.svelte";
 
   export let accountId = null;
+  let toAddress;
+  let amount;
+
+  const openSendModal = () => {
+    fromStore.set(accountId);
```

Comment:
There are two ways to do store updates with forms:

1. You can `set` the `store` upon form submission, so nothing is saved until the user decides to click `submit`. That's what is going on here - the store doesn't change until `openSendModal` is called.

2. You can `set` the `store` reactively every time the user changes the input value, as seen below in `SendFunds` (e.g. `$: amountStore.set(amount);`). With this approach, everything is automatically saved while the user types - see [docs on reactive declarations here](https://svelte.dev/tutorial/reactive-declarations)

Both are valid and have their pros and cons; it's a question of UX.